### ISSUE TITLE
timeit magic does not work when autoimport is enabled

### ISSUE THREAD

**Issue by tbodt** [anntzer/ipython-autoimport#1]
```
/usr/local/Cellar/python3/3.6.0_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/IPython/core/magic.py in <lambda>(f, *a, **k)
    186     # but it's overkill for just that one bit of state.
    187     def magic_deco(arg):
--> 188         call = lambda f, *a, **k: f(*a, **k)
    189
    190         if callable(arg):

/usr/local/Cellar/python3/3.6.0_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/IPython/core/magics/execution.py in timeit(self, line, cell)
   1053
   1054         ns = {}
-> 1055         exec(code, self.shell.user_ns, ns)
   1056         timer.inner = ns["inner"]
   1057

TypeError: exec() globals must be a dict, not AutoImportMap
```

**Comment by anntzer** [comment]
It looks like this is fixed as of master.  Can you confirm?  If it works for you as well I will make a new release "soon".

**Comment by tbodt** [comment]
It is indeed fixed.

**Comment by anntzer** [comment]
Released v0.2.
